vnet_test.go

```diff
@@ -529,16 +535,17 @@ func TestStreamClose(t *testing.T) {
 			venv.dropNextReconfigChunk(1)
 		}
 
-		// Immediately close the stream
+		// Wait server accept stream.
+		time.Sleep(time.Millisecond * 100)
```

Comment:
@enobufs Here I have another problem, if close the client stream Immediately, the server won't receive anything, it will block at server.AcceptStream(), block the test.
I think it's fine in the real world, do you think we should let DECP pass before closed?

Reply:
Please review this PR.  I simply reverted the changes to TestStreamClose.
https://github.com/jerry-tao/sctp/pull/4

Reply:
@enobufs  Thanks for your help, I was planning to fix CI when the PR is ready. 
I'm still testing it with browser, need one more day to make sure it worked as expected.
When the PR is ready, I will remove the WIP title.